### PR TITLE
python3.pkgs.liblzfse: init at 0.4.1

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119990` at 931c55a8 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.liblzfse</li>
    <li>python39Packages.liblzfse</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/liblzfse/default.nix:21:0:
```
   |
21 |     description = "Python bindings for LZFSE";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/liblzfse/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pkgs
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/liblzfse/default.nix:6]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/liblzfse/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  pname = "pyliblzfse";
+  version = "0.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "bb0b899b3830c02fdf3dbde48ea59611833f366fef836e5c32cf8145134b7d3d";
+  };
+
+  preBuild = ''
+    rm -rf lzfse
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/liblzfse/default.nix:16]:
```suggestion
    rm -r lzfse
```

File: pkgs/development/python-modules/liblzfse/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    ln -s ${pkgs.lzfse.src} lzfse
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/liblzfse/default.nix:17]:
```suggestion
    ln -s ${lzfse.src} lzfse
```

File: pkgs/development/python-modules/liblzfse/default.nix [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  '';
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/liblzfse/default.nix:19]: missing checkPhase and pythonImportsCheck.

File: pkgs/development/python-modules/liblzfse/default.nix [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pkgs
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/liblzfse/default.nix:4]:
```suggestion
, lzfse
```

**Pull request merged**